using-ide.md

```diff
@@ -138,7 +145,7 @@ html-based UI. For details see the [Observatory page](https://dart-lang.github.i
 To open Observatory:
 
 1. Run your app in debug mode.
-1. In the Debug panel at the bottom (see screenshot below), click the **Stopwatch icon** ('Open Observatory').
+1. Select 'open observatory from the In the Debug panel at the bottom (see screenshot below), click the **Stopwatch icon** ('Open Observatory').
```

Comment:
--> Select the 'Open Observatory' action from the Debug panel (see screenshot below).

Reply:
done